frontend/src/App.jsx

```diff
@@ -1,17 +1,16 @@
-import React from "react";
-import { useState } from "react";
+import React, { useState } from "react";
 import Navbar from "./components/Navbar/Navbar";
 import Corner from "./components/Corner/Corner";
 import Router from "./router";
 
 function App() {
-  const [searchValue, setSearchValue] = useState("");
-
+  const [inputValue, setInputValue] = useState("");
```

Comment:
there is no need to rename this

Reply:
Do I have to change anything here?
I changed the state name because the` inputValue, setInputValue` name is used mostly in the end components so in order to keep the state name the same in all components I changed the declarative name itself. Now the state name is the same in all the components.